demisto_sdk/validation/file_validator.py

```diff
@@ -256,15 +255,18 @@ def is_valid_release_notes(self, file_path):
     def validate_files(self, affected_files, is_added=False):
         REGEX_TO_VALIDATOR = {
```

Comment:
`StructureValidator` can be created here and pass to each of the functions.

Reply:
no need to put too much of effort on it
this written in a bad way
will refactor some of the code anyway